### ISSUE TITLE
It doesn't open without any clue.

### ISSUE THREAD

**Issue by SergaTV** [alexballas/go2tv#29]
I have downloaded latest release fo windows_amd64. extracted exe file, opened and nothing! Not even console appear, i also tried it run like cli in cmd, but it prints nothing and force close. 
My operating system is Windows 7 7601 Max SP1, amd athlon proccessor and 2gb ram. Maybe it needs more ram to load? It really didn't give me any log text or any clue to think about. Is it possile somehow fix this? i can provide any info need :
![cmd screenshot (maybe need)](https://user-images.githubusercontent.com/63511782/147067657-172d1d8e-31ba-49a0-9ef4-76dac8e2dbb2.png)

**Comment by SergaTV** [comment]
It seems to really because of not enough ram, I ran it on 16gb ram and now it works, lol.

**Comment by alexballas** [comment]
Hello, thanks for raising this.

I don't believe RAM is the issue here. I did manage to run it fine (consistently) on a 1 GB Windows 10 VM. It was slow, but it ran fine.

What you report seems to be related to the Windows 7 build which is outdated. The windows builds were only tested on Windows 10. I'll try to see if I can find a Windows 7 machine to test it on.

**Comment by SergaTV** [comment]
thanks

**Comment by alexballas** [comment]
I had a quick look at this one. Can you please share some more info on your Windows 7 installation? 
Is it a Virtual Machine? If yes, can you please "Enable 3D acceleration" since the app requires OpenGL 2.0 or later.

**Comment by SergaTV** [comment]
No, it is a real machine with Windows 7 SP1, and athlon(tm) 64 x2 dual core processor 6000+ 3.01GHz, 
2gb ram, 
64bit windows, 
Gpu is before gt 8800.

I can run it on new pc with windows 10 and 16gb ram as I mentioned upper.

**Comment by alexballas** [comment]
Can you please share the Opengl version on that machine? In the past I used apps like GLView to get that info.

**Comment by SergaTV** [comment]
GLInfo tells me that version is 1.1.0
[full_report_opengl.txt](https://github.com/alexballas/go2tv/files/7767704/full_report_opengl.txt)
this is full report from that program.

**Comment by alexballas** [comment]
Thanks. I believe this is the issue. You're using the bundled Opengl driver that comes with Windows. Try updating your graphics driver and test again.

I did a similar test with Virtual Box. Windows 7 with no 3D acceleration was behaving exactly as what you reported. It was dying with no error.
The GLInfo reported:
```
Vendor      : Microsoft Corporation
Renderer    : GDI Generic
Version     : 1.1.0
```

When I updated the setting to enabled the 3D Acceleration, the application started working fine and GLInfo was now reporting
```
Verdor : VMware, Inc.
Renderer: SVGA3D; build; RELEASE;
Version 2.1 Mesa mesa-17.3.9
```
